### ISSUE TITLE
Re-sizing columns in the player list panel ruins the column headers on windows

### ISSUE THREAD

**Issue by MaDDoXbr** [springlobby/springlobby#942]
Tested on Windows 10, SL 0.269. Check the anim gif here:
https://drive.google.com/open?id=16Z9FsP4OWnF_iLv8MRrokbDKo4ElCZON

**Comment by silentwings** [comment]
I can reproduce this, wierd...

**Comment by abma** [comment]
can you please test if this issue still present with the development version of springlobby?

atm latest:
https://springlobby.springrts.com/dl/develop/springlobby-0.269-150-g6501df858-win32.zip

**Comment by abma** [comment]
i can't reproduce this error on debian 10.

**Comment by MaDDoXbr** [comment]
You're right abma, the issue is really Windows related. If on the .exe properties you enable 'Disable Full Screen Optimizations' or don't set 'Hi DPI settings' to 'System (Enhanced)' it bugs out like in the gif I've added to the initial post. I'm attaching a picture of the best setting I've found which actually works.
![SL_windows_HiDPI_setting](https://user-images.githubusercontent.com/265085/72232424-dc480100-359f-11ea-9b49-381dab2867f2.jpg)

The important thing is that now it worked properly by default, which should be enough. Thanks for checking this out!
